TwitchChannelPointsMiner/classes/Twitch.py

```diff
@@ -244,6 +262,35 @@ def post_gql_request(self, json_data):
             )
             return {}
 
+    # Request for Integrity Token
+    # Twitch needs Authorization, Client-Id, X-Device-Id to generate JWT which is used for authorize gql requests
+    def post_integrity(self):
+        if (
+            datetime.now().timestamp() * 1000 - self.integrity_expire < 0
+            and self.integrity is not None
+        ):
+            return self.integrity
+        try:
+            response = requests.post(
+                GQLOperations.integrity_url,
+                json={},
+                headers={
+                    "Authorization": f"OAuth {self.twitch_login.get_auth_token()}",
+                    "Client-Id": CLIENT_ID,
+                    "User-Agent": self.user_agent,
+                    "X-Device-Id": self.device_id,
```

Comment:
Here twitch actually uses:
* Authorization
* Client-Id: kimne78kx3ncx6brgo4mv6wki5h1ko
* Client-Session-Id: <generated, 16 hex chars (0-9a-f)>
* Client-Version: <seems to be a fixed value, but changed maybe frequently>
* X-Device-Id: <generated, 32 alphanum (a-zA-Z0-9)

Reply:
- Add dynamically update `Client-Version` based on @RakSrinaNa suggestion [dae9c70](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/pull/594/commits/dae9c709862ea386e866ea83ef55bc0c70c5999f)
- Add generated `Client-Session-Id` [8594d54](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/pull/594/commits/8594d54f08eec723cc4bc238aa46412f4283df71)

Reply:
The client version looks like v4 UUID, maybe they change UUID version in the future so updating the regex will be easier